src/routes/docs/products/ai/tutorials/music-generation/+page.markdoc

```diff
@@ -0,0 +1,172 @@
+---
+layout: article
```

Comment:
the tutorial is missing from the side bar.

Reply:
added